src/components/TodoFilter.tsx

```diff
@@ -0,0 +1,54 @@
+import { useContext } from 'react';
+import { Status } from '../types/Status';
+
+import { TodoContext } from './TodosContext';
+import { TodoContextProps } from '../types/TodoContextProps';
+
+interface PropsFilter {
+  filter: Status;
+  setFilter: (filter: Status) => void;
+}
+
+export const TodosFilter: React.FC<PropsFilter> = () => {
+  const todoContext = useContext<TodoContextProps>(TodoContext);
+
+  const filter = todoContext?.filter || Status.ALL;
+  const setFilter = todoContext?.setFilter || (() => {});
+
+  return (
+    <nav className="filter" data-cy="Filter">
+      <a
+        href="#/"
+        className={filter === Status.ALL
+          ? 'filter__link selected'
+          : 'filter__link'}
```

Comment:
please use `classnames` library, like 

```
className={cn('filter__link', {
  selected: activeFilter === Status.ALL,
})}
```

don't forget to update in other filter links